streaming-binlog/src/test/java/org/openmrs/analytics/FhirConverterTest.java

```diff
@@ -92,6 +108,23 @@ public void shouldFetchFhirResourceAndUploadToFhirStore() throws IOException {
 		Mockito.verify(parquetUtil, Mockito.never()).write(Mockito.<Resource> any());
 	}
 	
+	@Test
+	public void shouldGetUuidFromParent() throws SQLException, PropertyVetoException, ClassNotFoundException {
+		
+		Map<String, String> messageBody = DebeziumTestUtil.genExpectedBody();
+		Map<String, Object> messageHeaders = DebeziumTestUtil.genExpectedHeaders(Operation.CREATE, "encounter");
+		String uuid = uuidUtil.getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString());
+		resource = new Encounter();
+		resource.setId(uuid);
```

Comment:
Where is this `resource` used?

Reply:
i have removed it 
@bashir2  kindly review